indexer/indexer_type.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+class IndexerType(Enum):
+    GITHUB = {"id_prefix": "github", "code_hosting_queue": "github", "database_table": "github"}
```

Comment:
`code_hosting_queue` is not used yet but it will be in the near future - they will be a queues for special `code hosting` analyzers (analyze of commits, PR, comments, etc.)